src/report-table/report-table.component.tsx

```diff
@@ -56,7 +56,16 @@ const ReportTable = (props: ReportTableProps) => {
 
   const getRequester = (
     performer: undefined | {display: string; reference: string},
-  ) => (performer && performer[0]?.display) || selfPatient
+  ) => {
+    if (performer) {
+      const lastIndex = performer[0]?.display?.lastIndexOf(' (')
```

Comment:
we can use regex pattern to replace.
`performer[0]?.display.replace(/\s\(.+\)/gm, "")`

Reply:
changes are made